src/runtime/hexagon/hexagon_device_api.h

```diff
@@ -50,6 +50,25 @@ class HexagonDeviceAPI final : public DeviceAPI {
   //! \brief Destructor
   ~HexagonDeviceAPI() {}
 
+  //! \brief Ensures resource managers are in a good state for the runtime
+  void AcquireResources() {
+    if (runtime_hexbuffs) {
+      LOG(INFO) << "runtime_hexbuffs has already been created";
```

Comment:
should this be an error, since it might be dirty now?

Reply:
I considered this, but I didn't want it to register as an error, as we don't enforce that all allocations are explicitly cleaned up.   (Several tests hit this.)

Reply:
This has been fixed - thanks for catching this.